.github/workflows/on_push_pr_no_fake_prerelease.yaml

```diff
@@ -0,0 +1,76 @@
+name: Push/PR no fake prereleases
+
+on:
+  workflow_call:
+    inputs:
+      original_repo_name:
+        required: true
+        type: string
+      tag:
+        required: false
+        type: string
+      run_static_analysis:
+        required: false
+        type: boolean
+        default: true
+      run_test_nix:
+        required: false
+        type: boolean
+        default: true
+      run_test_windows:
+        required: false
+        type: boolean
+        default: true
+      run_integration_nix:
+        required: false
+        type: boolean
+        default: true
+      run_test_build_nix:
+        required: false
+        type: boolean
+        default: true
+    secrets:
+      docker_hub_id:
+        description: "docker hub id"
+        required: false
+      docker_hub_password:
+        description: "docker hub password"
+        required: false
```

Comment:
why the other job was did not have the secrets?

Reply:
Same as above.